report/3.Theory/LanguageSpecification.tex

```diff
@@ -88,6 +100,22 @@ \subsection{Functions}
 
 The \texttt{<DclValue>} non-terminal refers to both \texttt{<Expr>} (expression) for type declarations and \texttt{<LambdaExpr>} (function parameters and body) for function declarations.
 
+The below listing showcases declarations in HCL using type inference as described above.
+The listing includes both assignment of expressions and assignment of lambda-expressions.
+
+\begin{lstlisting}[language=HCL,caption={Implicit type declarations in HCL.
+},firstnumber=1]
+var x = 3141596
```

Comment:
why unreasonable large numbers?
what about "5" and "3". two great numbers. the best.

Reply:
It's Pi and the Euler constant... sort of an mathmatical easter egg :D, but yeah sure it can be 5 and 3